docs/build/tutorials/platform/subnets/setup-dfk-node.md

```diff
@@ -0,0 +1,71 @@
+# Run a Node on DFK Subnet
+
+## Introduction
+
+We recently deployed the DFK subnet on Avalanche mainnet and successfully launched the [DFK chain](https://subnets.avax.network/defi-kingdoms/dfk-chain/explorer) on that subnet. This subnet is maintained by the [DeFi Kingdom](https://defikingdoms.com/)'s team and is based on Avalanche's [subnet-evm](https://github.com/ava-labs/subnet-evm).
+
+A subnet, or subnetwork, is a dynamic set of validators working together to achieve consensus on the state of a set of blockchains. Each blockchain is validated by exactly one subnet. A subnet can validate many blockchains. A node may be a member of many subnets. You can learn more about subnets [here](../subnets/README.md).
+
+## Requirement
+
+There is a requirement to run a node on the DFK subnet - 
+
+* You must be running a node on the Avalanche Primary Network. You can use this [guide](../../nodes-and-staking/set-up-node-with-installer) to set up your node using the installer script.
+* You are using the same system where you have deployed your mainnet node.
```

Comment:
It isn't exactly clear to me what the second bullet point says. Did you mean to say that the DFK subnet needs to be deployed where the mainnet node is deployed?

Reply:
Also, third bullet point might be:
* mainnet node needs to be bootstrapped completely before running the subnet